src/pkg/resourceviews/resourcetypes.go

```diff
@@ -114,10 +122,19 @@ func (r *ResourceTypeListView) Update() error {
 		}
 	}
 
+	// Add the resource types to the list
+	r.ResourceTypes = &[]ResourceType{}
+
 	for resourceType := range resourceTypes {
-		r.List.AddItem(resourceType, "", 0, func() {
-			r.SelectItem(resourceType)
-		})
+		rt := ResourceType{
+			Name:         resourceType,
+			ReadableName: strings.TrimPrefix(resourceType, "Microsoft."),
+			SelectedFunc: func() {
+				r.SelectItem(resourceType)
```

Comment:
Finally found it:
Looks like if you pass resourceType go actually takes a reference to the array of some sort. It works if you make a name variable, set that to resourceType and then set name in the function